platform/ios/src/MGLMapViewDelegate.h

```diff
@@ -308,6 +308,17 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (void)mapView:(MGLMapView *)mapView didChangeUserTrackingMode:(MGLUserTrackingMode)mode animated:(BOOL)animated;
 
+/**
+ Returns a CGPoint value to offset the user location annotation, relative to the
+ map view’s origin after applying the map view’s content insets.
+ 
+ This method will offset the user location annotation by the specified
+ X and Y values determined by the CGPoint.
```

Comment:
What is the default behavior when this method goes unimplemented? How does this method interact with the `MGLMapView.userLocationVerticalAlignment` property when it is set and this method is also implemented?